aiotfm/client.py

```diff
@@ -1078,6 +1082,17 @@ def is_tribe(tc, packet):
 				raise CommunityPlatformError(118, result)
 		return Tribe(packet)
 
+	async def getRoomList(self, gamemode):
+		"""|coro|
+		Get the room list
+
+		:param gamemode: Optional[:class:`aiotfm.enums.GameMode`] the room's gamemode.

```

Comment:
The function's parameter doesn't look optional to me but the description says so 🤔

Reply:
Oops I forgot to add the behavior